src/components/Tabs/Tabs.scss

```diff
@@ -0,0 +1,68 @@
+%tab {
+    background-color: var(--color-neutral-25);
+    border: {
+        color: var(--color-neutral);
+        style: solid;
+        top-left-radius: var(--border-radius);
+        top-right-radius: var(--border-radius);
+        width: var(--line-normal);
+        bottom-width: 0;
+    }
+    color: var(--link-color-normal);
+    cursor: pointer;
+    margin: {
+        top: var(--margin);
+        right: var(--margin);
+        left: var(--margin);
+    };
+    outline: none;
+    padding: var(--spacing-2x) var(--spacing-4x);
+    line-height: var(--line-height-normal);
+    text-decoration: none;
+    transition: border-color var(--transition-duration);
+
+    &:hover {
+        border-color: var(--color-neutral-60);
+    }
+}
+
+.Tabs {
+    color: black;
+    font: {
+        family: var(--font-family-primary);
+        size: var(--font-size-normal);
+    }
+
+    &__container {
+        border-bottom: {
+            color: var(--color-neutral);
+            style: solid;
+            width: var(--line-normal);
+        }
+        display: flex;
+        justify-content: flex-start;
+        padding: {
+            top: var(--spacing-2x);
+            left: var(--spacing-2x);
+            right: var(--spacing-2x);
+        }
+    }
+    &--justified {
+        .Tabs__container {
+            justify-content: space-between;
+        }
+    }
```

Comment:
Yes I added this prop initially but then chose to remove it because we could still add it once needed, but forgot to remove the corresponding class modifiers